security/pfSense-pkg-acme/Makefile

```diff
@@ -0,0 +1,97 @@
+# $FreeBSD$
+
+PORTNAME=	pfSense-pkg-acme
+PORTVERSION=	0.1
+CATEGORIES=	security
+MASTER_SITES=	# empty
+DISTFILES=	# empty
+EXTRACT_ONLY=	# empty
+
+MAINTAINER=	PiBa-NL
+COMMENT=	pfSense package acme
+
```

Comment:
Add LICENSE=	APACHE20 between COMMENT and USE_PHP leaving a blank line before and after